spell.ignore

```diff
@@ -0,0 +1,399 @@
+mux
+env
+merchantability
+lmr
+redhat
+rhel
+distro
+rmoura
+libc
+runtime
+stderr
+ctrl
+stty
+paginator
+usr
+conf
+config
+datadir
+dir
+api
+iso
+gpg
+vm
+args
+os
+jeos
+logdir
+rtype
+basestring
+tmp
+cleber
+cli
+cmd
+prepost
+unittests
+ascii
+unicode
+fixme
+html
+json
+sysinfo
+pre
+xunit
+params
+argparse
+urls
+whitespace
+shortname
+url
+str
+traceback
+filename
+namespace
+avocadolib
+docstring
+exc
+tuples
+tuple
+klass
+uid
+areis
+todo
+attr
+muxes
+nics
+sys
+stdout
+splitlines
+throbber
+mem
+func
+fmt
+itertable
+iterable
+ui
+subcommands
+subparsers
+utils
+hostname
+username
+autodetect
+pem
+uptime
+apahim
+proc
+sigterm
+subprocess
+sigstop
+sigtstp
+crosa
+ast
+realname
+bool
+autotest
+py
+intree
+ini
+unittesting
+sigsegv
+booleans
+distros
+logf
+gzipped
+basedir
+profiler
+mbligh
+apw
+shadowen
+mytest
+aaa
```

Comment:
@ldoktor, this is the comment:
~~~
 93         # 001-mytest;aaa
 94         # 001-mytest;a
 95         # 001-myte;aaa
~~~
How do you suggest it to be?